### DISCUSSION TITLE
GoPro QR standard?

### DISCUSSION THREAD

**Discussion by thelifeaquatic** [gopro/labs#351] [General]
Hi there,
Is there a "standard" that the GoPro QR tags are based on? Or is the source code used to generate them available? We want to use the tags to set camera settings/file names etc, but we have another piece of equipment (essentially a logger), that we want to be able to read the same code (using a barcode scanner and Pi Zero) to set the csv file name to match the GoPro.
 

Many thanks

**Comment by dnewman-gpsw** [comment]
QR Codes are standardized, any reader or writer can be used.  Is that your question?

**Reply by thelifeaquatic** [reply]
yes, thanks

**Reply by dnewman-gpsw** [reply]
So the question is answered.  I think.
